extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -116,12 +116,15 @@ export class AmpAnalytics extends AMP.BaseElement {
 
     /** @private {?IframeTransport} */
     this.iframeTransport_ = null;
+
+    /** @private {!boolean} */
```

Comment:
`{boolean}`

Reply:
I was curious why there is a !, copied from the line for this.isSandbox_

Reply:
I would expect more uses of isInabox checks in the future (I could be wrong) for taking different behavior for inabox or fast-fetch. That's why I feel that a separate field might be good. In addition for me to get familiar with the work flow.

Reply:
See https://google.github.io/styleguide/jsguide.html#jsdoc-nullability